frontend/app/locales/de.json

```
@@ -0,0 +1,167 @@
+  "blockingDuration.permanently": "Permanently",
+  "blockingDuration.week": "For a week",
+  "comment.block": "Block",
+  "comment.block-user": "Do you want to block {userName} {duration}?",
+  "comment.blocked-user": "Blocked",
+  "comment.blocking-period": "Blocking period",
+  "comment.cancel": "Cancel",
+  "comment.controversy": "Controversy: {value}",
+  "comment.copied": "Copied!",
+  "comment.copy": "Copy",
+  "comment.delete": "Delete",
+  "comment.delete-message": "Do you want to delete this comment?",
+  "comment.deleted-comment": "This comment was deleted",
+  "comment.deleted-user": "Deleted",
+  "comment.edit": "Edit",
+  "comment.expired-time": "Editing time has expired.",
+  "comment.go-to-parent": "Go to parent comment",
+  "comment.hide": "Hide",
+  "comment.hide-user-comment": "Do you want to hide comments of {userName}?",
+  "comment.pin": "Pin",
+  "comment.pin-comment": "Do you want to pin this comment?",
+  "comment.reply": "Reply",
+  "comment.time": "{day} at {time}",
+  "comment.toggle-verification": "Toggle verification",
+  "comment.unblock": "Unblock",
+  "comment.unblock-user": "Do you want to unblock this user?",
+  "comment.unpin": "Unpin",
+  "comment.unpin-comment": "Do you want to unpin this comment?",
+  "comment.unverified-user": "Unverified user",
+  "comment.unverify-user": "Do you want to unverify {userName}?",
+  "comment.verified-user": "Verified user",
+  "comment.verify-user": "Do you want to verify {userName}?",
+  "comment.vote-error": "Voting error: {voteErrorMessage}",
+  "commentForm.exceeded-size": "{fileName} exceeds size limit of {maxImageSize}",
+  "commentForm.input-placeholder": "Your comment here",
+  "commentForm.new-comment": "New comment",
+  "commentForm.notice-about-styling": "Styling with <a>Markdown</a> is supported",
+  "commentForm.preview": "Preview",
+  "commentForm.reply": "Reply",
+  "commentForm.save": "Save",
+  "commentForm.send": "Send",
+  "commentForm.subscribe-by": "Subscribe by",
+  "commentForm.subscribe-or": "or",
+  "commentForm.unexpected-error": "Something went wrong. Please try again a bit later.",
+  "commentForm.upload-file-fail": "{fileName} upload failed with \"{errorMessage}\"",
+  "commentForm.uploading": "Uploading...",
+  "commentForm.uploading-file": "uploading {fileName}...",
+  "commentSort.sort-by": "Sort by",
+  "commentsSort.best": "Best",
+  "commentsSort.least-controversial": "Least controversial",
+  "commentsSort.least-recently-updated": "Least recently updated",
+  "commentsSort.most-controversial": "Most controversial",
+  "commentsSort.newest": "Newest",
+  "commentsSort.oldest": "Oldest",
+  "commentsSort.recently-updated": "Recently updated",
+  "commentsSort.worst": "Worst",
+  "emailLoginForm.back": "Back",
+  "emailLoginForm.confirm": "Confirm",
+  "emailLoginForm.email-address": "Email Address",
+  "emailLoginForm.empty-token": "Token field must not be empty",
+  "emailLoginForm.expiredToken": "Token is expired",
```

Looks like it should be:
```suggestion
  "emailLoginForm.expired-token": "Token is expired",
```